packages/paste-design-tokens/tokens/themes/console/global/background-color.yml

```diff
@@ -83,6 +86,9 @@ props:
   color-background-required:
     value: "{!palette-red-50}"
     comment: Required background color
+  color-background-new:
+    value: "{!palette-purple-20}"
+    comment: Background color for a badge indicating a new status.
```

Comment:
This comment should be changed too hahah

Reply:
Right! Good catch. Got it.